### PR TITLE
Create ScamsEducationIllustration

### PR DESCRIPTION
**Pull request by nickoola**
## Description

Create ScamsEducationIllustration

## How it was tested

Local environment with ui-library

Screenshot 2021-03-31 at 11 03 54

### CODE REVIEW

File: src/illustration/scamsEducation.tsx [hunk 1]
```diff
@@ -0,0 +1,324 @@
+import React from 'react'
+
+import { Illustration, IllustrationProps } from '../_utils/illustration'
+
+export const ScamsEducationIllustration = ({ title }: IllustrationProps): JSX.Element => (
+  <Illustration title={title}>
+    <mask
+      id="mask0"
```
Comment (mcellier) [inline src/illustration/scamsEducation.tsx:8]: avoid generic ids like this, we might get duplicates between illustrations.

**Review by mcellier — APPROVED** [review]
Is it just me or is it weirdly centered on the right?

File: src/illustration/scamsEducation.tsx [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+        fill="#FACFC4"
+      />
+      <path
+        d="M60.7723 82.731L76.4235 87.3536V101.498C76.4235 102.334 73.7746 108.525 68.8987 108.753H68.7051C68.6568 108.758 68.6082 108.758 68.5599 108.753C68.5262 108.764 68.4898 108.764 68.4561 108.753C68.2261 108.774 67.9945 108.774 67.7645 108.753C61.1942 108.256 60.7446 101.705 60.7446 101.705L60.7723 82.731Z"
+        fill="#FACFC4"
+      />
+      <g opacity="0.18">
```
Comment (Tuxosaurus) [inline src/illustration/scamsEducation.tsx:200]: Do the other illustrations have "opacity" too? I would rather avoid that.

File: src/illustration/scamsEducation.tsx [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+        d="M85.9541 156.002L87.2198 135.059L97.7738 135.695L96.515 156.638L85.9541 156.002ZM87.7246 135.625L86.5212 155.505L96.0102 156.072L97.2066 136.192L87.7246 135.625Z"
+        fill="white"
+      />
+      <path
+        d="M91.716 148.381C89.2123 148.229 87.3312 146.896 87.4211 145.341C87.5179 143.786 89.5443 142.687 92.048 142.839C94.5516 142.991 96.4328 144.325 96.336 145.88C96.2825 146.301 96.1179 146.7 95.859 147.036C95.6001 147.373 95.2563 147.634 94.8628 147.794C94.339 148.057 93.7762 148.234 93.196 148.319C92.706 148.388 92.2102 148.409 91.716 148.381ZM87.9605 145.375C87.8844 146.585 89.6204 147.718 91.7436 147.842C92.7302 147.927 93.7214 147.746 94.6138 147.317C94.9235 147.197 95.1963 146.998 95.405 146.74C95.6137 146.481 95.7511 146.173 95.8034 145.845C95.8726 144.636 94.1435 143.503 92.0134 143.372C91.5652 143.343 91.1152 143.364 90.6716 143.434C89.157 143.662 88.0158 144.415 87.9605 145.375Z"
+        fill="white"
+      />
+      <path
+        d="M94.2606 140.642V141.554H93.569V141.333L90.4844 141.243V140.552L94.2606 140.642Z"
+        fill="white"
```
Comment (Tuxosaurus) [inline src/illustration/scamsEducation.tsx:292]: nit: Should we replace all "white" with "#FFF" for consistency?

Comment (mcellier) [reply src/illustration/scamsEducation.tsx:292]: Or `{color.white}`

**Review by pith — APPROVED** [review]
Are we sure they won't be naked?

**Comment by nickoola** [comment]
> Is it just me or is it weirdly centered on the right?

After re-export from the right layer on Figma, it fixed the centered issue?

**Comment by Tuxosaurus** [comment]
> Are we sure they won't be naked?

Should be alright, they have plain color clothing ;)

**Comment by nickoola** [comment]
> Are we sure they won't be naked?

I tested the storybook with Chrome/Safari/Firefox to be sure. So I think it's ok 😄

**Pull request merged**